CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#8912](https://github.com/rubocop-hq/rubocop/pull/8912): Fix Layout/ElseAlignment for rescue/else/ensure inside do/end blocks with assignment. ([@miry][])
```

Comment:
Can you enclose the cop name in back quotes?

```suggestion
* [#8912](https://github.com/rubocop-hq/rubocop/pull/8912): Fix `Layout/ElseAlignment` for rescue/else/ensure inside do/end blocks with assignment. ([@miry][])
```

And, can you rebase with the latest mater branch or use the new changelog option? (It can be updated by either one.)
https://github.com/rubocop-hq/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format